api/src/main/java/org/pmiops/workbench/rdr/RdrExportServiceImpl.java

```diff
@@ -183,7 +183,11 @@ public void exportWorkspaces(List<Long> workspaceIds, boolean backfill) {
         // data already from a backfill, we'd still want to resend any normal modifications, if any,
         // in order to trigger this review process.
         if (!backfill) {
-          updateDbRdrExport(RdrEntity.WORKSPACE, workspaceIds);
+          updateDbRdrExport(
```

Comment:
could you explain how this change resolves the problem?

Reply:
175 line `toRdrWorkspace` is where FC calls happens. If it throws exception, it would return null, so not be populated into `rdrWorkspacesList` list. So rdrWorkspacesList should be the source to update with DB.

Reply:
ok, thanks!  Could you add a few things to make this more clear?

* add a `@Nullable` annotation to toRdrWorkspace()
* add a comment to line 176 like "skip failures and continue"
* move the log "successfully exported workspace data" inside the `rdrWorkspacesList.isEmpty()` check
* update the log message to include only the successful output